### PR TITLE
Add new Debian 8.4 image

### PR DESCRIPTION
**Pull request by mellinoe**
@mmitche This adds a newer Debian 8.4 image. I have several more of these coming soon.

### CODE REVIEW

File: management/nodes/Images.csv [hunk 1]
```diff
@@ -1,14 +1,16 @@
-# Temporary.  List of images names, the storage account sub-path for each one, and the list of storage accounts that
```
Comment (mellinoe) [inline management/nodes/Images.csv]: Excel saved the file like this; I can revert if you'd like.

**Comment by mellinoe** [comment]
I updated Utilities.groovy as well; they are now listed in ci.dot.net/configuri.

File: jobs/generation/Utilities.groovy [hunk 2]
```diff
@@ -302,11 +302,11 @@ class Utilities {
                                 ],
                             'Debian8.4' :
                                 [
-                                '' : 'auto-deb84-20160623',
+                                '' : 'deb84-20170214',
```
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: Can you remove the empty-string version?

**Comment by mmitche** [comment]
LGTM aside from minor nit.

**Pull request merged**